python/ray/serve/api.py

```diff
@@ -573,14 +573,15 @@ def run(
         route_prefix=route_prefix,
         logging_config=logging_config,
     )
+    logger.warning("Deployed app successfully.")
```

Comment:
```suggestion
    logger.warning(f"Deployed app '{name}' successfully.")
```

Reply:
this should be `INFO`-level

Reply:
Due to this not using serve logger, info level won't log it to stdout 😅